l1-contracts/contracts/bridge/L1NativeTokenVault.sol

```diff
@@ -220,11 +220,12 @@ contract L1NativeTokenVault is IL1NativeTokenVault, IL1AssetHandler, Ownable2Ste
 
     ///  @inheritdoc IL1AssetHandler
     function bridgeRecoverFailedTransfer(
+        address _depositSender,
```

Comment:
Is it ok if you put depositSender last (i.e. in third place?) the chainId is always the first in bridgeBurn, bridgeMint